diff --git a/config/catamel/component-config.local.json b/config/catamel/component-config.local.json
new file mode 100644
index 0000000..7218015
--- /dev/null
+++ b/config/catamel/component-config.local.json
@@ -0,0 +1,41 @@
+{
+    "loopback-component-explorer": {
+        "mountPath": "/explorer"
+    },
+    "loopback-component-visualizer": {
+        "mountPath": "/visualize"
+    },
+    "loopback-component-model-diagram": {
+        "mountPath": "/modeldiagram"
+    },
+    "../node_modules/loopback-component-mq/lib": {
+        "options": {
+            "restPort": 15672,
+            "acls": [{
+                "accessType": "*",
+                "principalType": "ROLE",
+                "principalId": "$unauthenticated",
+                "permission": "DENY"
+            }]
+        },
+        "topology": {
+            "connection": {
+                "uri": "amqp://admin:password@scicatlive-local-rabbitmq-1"
+            },
+            "exchanges": [{
+                "name": "jobs.write",
+                "type": "topic",
+                "persistent": true
+            }],
+            "queues": [{
+                "name": "client.jobs.write",
+                "subscribe": true
+            }],
+            "bindings": [{
+                "exchange": "jobs.write",
+                "target": "client.jobs.write",
+                "keys": ["jobqueue"]
+            }]
+        }
+    }
+}
\ No newline at end of file
diff --git a/config/catamel/config.local.js b/config/catamel/config.local.js
index b393a0a..1207d45 100644
--- a/config/catamel/config.local.js
+++ b/config/catamel/config.local.js
@@ -21,15 +21,26 @@ module.exports = {
     jobError: "Finished with errors",
     jobCancel: "Cancelled",
   },
-  smtpSettings: {
+  // Add to this section if you want to use RabbitMQ for the Proposals mechanism
+  /*rabbitmq: {
+    enabled: false,
+    host: null,
+    port: null,
+    queue: null
+  },*/
+  /*smtpSettings: {
     host: "SMTP.YOUR.DOMAIN",
     port: 25,
     secure: false,
-  },
+  },*/
   smtpMessage: {
     from: "scicatarchivemanager@YOUR.DOMAIN",
     to: undefined,
     subject: "[SciCat " + process.env.NODE_ENV + "]",
     text: undefined, // can also set html key and this will override this
   },
+  queue: "rabbitmq",
+  logbook: {
+    enabled: false
+  }
 };
diff --git a/config/frontend/config.json b/config/frontend/config.json
index c5121f2..b366759 100644
--- a/config/frontend/config.json
+++ b/config/frontend/config.json
@@ -17,7 +17,7 @@
    "jsonMetadataEnabled":true,
    "jupyterHubUrl":"",
    "landingPage":"",
-   "lbBaseURL":"http://127.0.0.1",
+   "lbBaseURL":"http://catamel.localhost",
    "localColumns":[
       {
          "name":"datasetName",
diff --git a/docker-compose.yaml b/docker-compose.yaml
index 8404511..c11f98a 100644
--- a/docker-compose.yaml
+++ b/docker-compose.yaml
@@ -34,9 +34,11 @@ services:
       - ./config/catamel/config.local.js:/usr/src/app/server/config.local.js
       - ./config/catamel/datasources.json:/usr/src/app/server/datasources.json
       - ./config/catamel/providers.json:/usr/src/app/server/providers.json
+      - ./config/catamel/component-config.local.json:/usr/src/app/server/component-config.local.json
     labels:
       - "traefik.http.routers.catamel.rule=PathPrefix(`/api`, `/auth`, `/explorer`)"
       - "traefik.http.routers.catamel.entrypoints=web"
+      - "traefik.http.routers.catamel.rule=Host(`catamel.localhost`)"
   catanie:
     #image: dacat/catanie:latest
     image: scicatproject/frontend:v4.0.0
@@ -48,6 +50,7 @@ services:
     labels:
       - "traefik.http.routers.catanie.rule=PathPrefix(`/`)"
       - "traefik.http.routers.catanie.entrypoints=web"
+      - "traefik.http.routers.catanie.rule=Host(`catanie.localhost`)"
   search-api:
     image: dacat/panosc-search-api:latest
     depends_on: 
